examples/java-matter-controller/java/src/com/matter/controller/commands/common/IPAddress.java

```diff
@@ -0,0 +1,16 @@
+package com.matter.controller.commands.common;
+
+import java.net.InetAddress;
+
+public final class IPAddress {
+  private InetAddress address;
```

Comment:
Can this be marked @Nullable? And also handle this in the toString() method? Since the constructor doesn't require a value the default is null. If possible, I would add a constructor which takes in an InetAddress (then you don't need @Nulalble) but if it's not known at construction time then it should be nullable.